bin/download-tgz

```diff
@@ -17,11 +17,20 @@ const commands = require('../lib/commands');
 
 program.version(packageJson.version);
 
+const parseConcurrency = (value) => {
```

Comment:
we need a default concurrency here, I would suggest `require('os').cpus().length`

Reply:
Do you mean to default the concurrency option value like the following?
```js
.option('-c, --concurrency <concurrency>', 'number of concurrent download', commands.parseConcurrency, cpuCount)
```

Reply:
yes, something like that